wasm2map/src/lib.rs

```diff
@@ -415,7 +434,19 @@ impl WASM {
             .filter_map(|p| Some(p.as_os_str().to_str()?.to_owned()))
             .collect::<Vec<_>>();
 
-        (mappings, source_paths)
+        let contents = bundle.then(|| {
+            source_paths
+                .iter()
+                .map(Path::new)
+                .map(|path| {
+                    fs::read_to_string(path)
+                        .map(|content| Cow::Owned(format!(r#""{}""#, Self::json_encode(&content))))
```

Comment:
Please update `WASM::` to `Self::` in this file as well. Consistency is important for readability and `Self` seems to be the popular approach across the rustverse.

Reply:
Done.
I also added `#![warn(clippy::use_self)]`.